Core_OverlayMods/Clothes/KoiClothesOverlayGui.cs

```diff
@@ -100,14 +100,29 @@ void ReadTex(string texturePath)
             }
         }
 
-        private static void RefreshInterface(string category = null)
+        internal static void RefreshInterface(string category = null)
         {
             if (!MakerAPI.InsideMaker || _refreshInterfaceRunning || _refreshInterface == null) return;
 
             _refreshInterfaceRunning = true;
             _instance.StartCoroutine(RefreshInterfaceCo(category));
         }
 
+        internal static void RefreshMenuColors(int parts)
+        {
+            if (!MakerAPI.InsideMaker) return;
+
+#if KK || KKS || EC
+            var topObject = GameObject.Find("CustomScene/CustomRoot/FrontUIGroup/CustomUIGroup/CvsMenuTree");
```

Comment:
Would prefer not using global `GameObject.Find` since it's prone to failing if any of the objects are disabled for whatever reason by another plugin.